google/cloud/security/inventory/inventory_loader.py

```diff
@@ -173,38 +176,62 @@ def _build_pipelines(cycle_timestamp, configs, **kwargs):
 
     Raises: inventory_errors.LoadDataPipelineError.
     """
+
     pipelines = []
-    crm_v1_api_client = crm.CloudResourceManagerClient()
-    crm_v2beta1 = crm.CloudResourceManagerClient(version='v2beta1')
-    gcs_api_client = gcs.StorageClient()
-    compute_api_client = compute.ComputeClient()
 
+    # Commonly used clients, we'll stuff these in a variable for re-use.
```

Comment:
nit: it would be nice if you can add a comment that this is so that we can re-use the rate-limiter?